dll/win32/shell32/wine/control.c

```diff
@@ -1016,8 +1016,11 @@ static	void	Control_DoLaunch(CPanel* panel, HWND hWnd, LPCWSTR wszCmd)
                 else if (!pchSecondComma) 
                     pchSecondComma = &wszCmd[i];
                 break;
-            case L' ': 
-                pchLastUnquotedSpace = &wszCmd[i];
+            case L' ':
+                if (!pchFirstComma)
+                    pchFirstComma = &wszCmd[i];
```

Comment:
Is this needed because the "first" comma (e.g. in `desk.cpl,,1`) could actually be replaced by a space?

Reply:
Yes, this is the case for the `desk.cpl desk,@Appearance` syntax actually. `desk` is the dialogbox name, `@Appearance` is the extra params.